esp/src/src/Utility.ts

```diff
@@ -1181,4 +1181,14 @@ export function wrapStringWithTag(string, tag = "span") {
 
 export function isSpill(sourceKind: string, targetKind: string): boolean {
     return sourceKind === "2" || targetKind === "71";
-}
\ No newline at end of file
+}
+
+export function convertToLocalTime(dateString) {
+    const modifiedDate = new Date(dateString);
+    return modifiedDate.toLocaleString();
```

Comment:
These conversion functions may have to be slightly more complicated unfortunately. Using the `Date.toLocaleString()` and `.toUTCString()` functions as is would also change the formatting of the timestamps from something like `2023-07-21 01:14:44` to `Fri, 21 Jul 2023 01:14:44 GMT`. Which might be an improvement, actually... But just something to consider.